src/pages/Validators/Index.tsx

```diff
@@ -30,23 +30,63 @@ export default function ValidatorsPage() {
         <>
           <ValidatorsMap />
           {inDev && (
-            <Tabs value={onDelegatory} onChange={handleTabChange}>
-              <Tab label="All Nodes" value={false} />
-              <Tab label="Delegation Nodes" value={true} />
-            </Tabs>
+            <ToggleButtonGroup
+              sx={{
+                width: "100%",
+                "@media screen and (min-width: 40em)": {
+                  width: "40%",
+                },
+              }}
+              fullWidth={true}
+              size="large"
+              value={onDelegatory}
+              onChange={handleTabChange}
+              exclusive={true}
+            >
+              <ToggleButton value={false}>
+                <Typography sx={{textTransform: "capitalize"}}>
+                  All Nodes
+                </Typography>
+              </ToggleButton>
+              <ToggleButton value={true}>
+                <Typography sx={{textTransform: "capitalize"}}>
+                  Delegation Nodes
+                </Typography>
+              </ToggleButton>
+            </ToggleButtonGroup>
           )}
           <Box sx={{width: "auto", overflowX: "auto"}}>
             <ValidatorsTable onDelegatory={onDelegatory} />
           </Box>
         </>
       ) : (
         <>
-          {inDev ? (
-            <Tabs value={onDelegatory} onChange={handleTabChange}>
-              <Tab label="All Nodes" value={false} />
-              <Tab label="Delegation Nodes" value={true} />
-            </Tabs>
-          ) : null}
+          {inDev && (
```

Comment:
Also to avoid code duplication, consider to change the logic structure. For example:

```
const validatorsTable = isMainnet ? <ValidatorsTable /> : <OldValidatorsTable />;

return inDev ? <ValidatorsTabs /> : validatorsTable;
```

Then inside `ValidatorsTabs`:
```
const validatorsTable = isMainnet ? <ValidatorsTable /> : <OldValidatorsTable />;

// the following is not real code. it simply means two tables under two tabs
return <Tabs>
  {validatorsTable}
  <DelegatoryValidatorsTable />
</Tabs>
```

Then inside `DelegatoryValidatorsTable`, this is the actual table component for delegatory validators. And this table will be shared between mainnet/testnet/devnet.

Of course this is only one of the ways that we can implement it. Please let me know if you have questions.